runbook.md

```diff
@@ -0,0 +1,17 @@
+# Runbook for proposed changes with steps
+
+## -- changes:
+ 1. secure access to prod trust claim DB
+ 2. change digital ocean droplet to test server with a separate DB for trust claim
+ 3. maintain digital ocean for Taiga and Aws for trust claim
+ 4. set-up CI/CD on dev (digital ocean trustclaim server)
+ 5. rewrite taiga.whatscookin.us to redirect to Taiga.linkedtrust.us
```

Comment:
sure this is fine you can redirect Taiga now actually - fyi is a good idea to group together the related steps, like the taiga in one group, and the linkedtrust in another, so we do it in chunks that relate to each other.  but this is fine.

Reply:
this changes has been effected, also should i terminate the ec2 in aws that's running taiga, since it's not being used anymore, and it's basically incurring costs

Reply:
yes, go ahead